ui/tests/acceptance/clients/counts/acme-test.js

```diff
@@ -44,9 +44,17 @@ module('Acceptance | clients | counts | acme', function (hooks) {
     this.nsPath = 'ns1';
     this.mountPath = 'pki-engine-0';
     this.expectedValues = {
+      // since there are no nested counts under ns1, this calculation is accurate
       nsTotals: byNamespace.find((ns) => ns.label === this.nsPath),
-      nsMonthlyUsage: byMonth.map((m) => m?.namespaces_by_key[this.nsPath]).filter((d) => !!d),
-      nsMonthActivity: byMonth.find(({ month }) => month === '9/23').namespaces_by_key[this.nsPath],
+      nsMonthlyUsage: byMonth
+        .map((m) => {
+          // loosely mocks what is returned when the namespace header is passed
+          const base = { month: m.month, timestamp: m.timestamp, label: m.label };
+          return (
```

Comment:
This is gross 😢 I couldn't use the filter function I use in the mirage handler because the data is already normalized. I could instead read the data that is calculated from the controller after namespace filtering is applied, but that felt like it wouldn't actually test anything (especially without controller tests _yet_)
Another thing I could do is hardcode the expectations, which wouldn't be very hard since the mock data is static not generated.